reports.py

```diff
@@ -0,0 +1,175 @@
+from reportlab.lib.pagesizes import letter
+from reportlab.pdfgen import canvas
+from datetime import datetime
+
+
+class Report():
+    def __init__(self):
+        """Placeholder. Need help to put stuff here"""
+
+    def calculate_pages(self):
+        """
+        Simulated for loop counter (for total page count)
+        """
+        fake_y_pos = 615
+        total_pages = 1
+        for i in range(len(data['names'])):
```

Comment:
This should be an argument instead

Reply:
This is just a counter. There might be a better way but I haven't found it. Will try to refactor as soon as I can.